databases/week3/food_reservation.sql

```diff
@@ -0,0 +1,73 @@
+CREATE DATABASE share_meals
+    DEFAULT CHARACTER SET = 'utf8mb4';
+
+use share_meals;
+SET NAMES utf8mb4;
+
+CREATE Table `Meal`(
+    `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `title` varchar(255) NOT NULL,
+    `description` text,
+    `location` varchar(255) NOT NULL,
+    `when` DATETIME NOT NULL,
+    `max_reservations` int(10),
+    `price` decimal(10,3) NOT NULL,
+    `created_date` DATE,
+    PRIMARY KEY (`id`)
+);
+
+CREATE TABLE `Reservation`(
+    `id` int(10) NOT NULL AUTO_INCREMENT,
+    `number_of_guests` int(10),
+    `meal_id` int(10) UNSIGNED NOT NULL,
+    `created_date` DATE NOT NULL,
+    `contact_phonenumber` varchar(100),
+    `contact_name` varchar(255),
+    `contact_email` varchar(255),
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`) ON DELETE CASCADE
```

Comment:
If we delete all reservations for a given meal, the meal entry will also be deleted, is that what you want? (It is ok, just checking ;) )